WasatchNET/Driver.cs

```diff
@@ -154,95 +173,99 @@ public int openAllSpectrometers()
             // Add Wasatch Photonics SPI spectrometers
             ////////////////////////////////////////////////////////////////////
 
-            if (spectrometers.Count == 0)
+            if (Environment.GetEnvironmentVariable("WASATCHNET_USE_SPI") != null)

```

Comment:
I wasn't sure whether to leave this in or not.  I think it could be useful to be able to make the SPI and SeaBreeze stuff totally configurable, so people who aren't using it (which is the vast majority of customers) don't see it at all.  But it's debatable whether it should be opt-in or opt-out, and the mechanism for either.  I'm open to discussion on this, and don't want to impose hardships on Production either.

Reply:
Yeah.  Basically, on Production machines, we'd just have a one-time process where we go into My Computer -> Properties -> Environment variables, define those two variables, and we're done.  They'll stay defined computer-wide until someone explicitly deletes them.